airflow/api_connexion/endpoints/user_endpoint.py

```diff
@@ -119,7 +119,7 @@ def post_user() -> APIResponse:
     user = security_manager.add_user(role=roles_to_add, **data)
     if not user:
         detail = f"Failed to add user `{username}`."
-        return Unknown(detail=detail)
+        raise Unknown(detail=detail)
```

Comment:
Oops, a bug found?

Reply:
Yeah. Fixed yesterday. Rebasing would fix it